DashAI/back/config_object.py

```diff
@@ -17,6 +17,9 @@
         dashai_path, "back/dataloaders/params_schemas/"
     ),
     SquemaTypes.task: os.path.join(dashai_path, "back/tasks/tasks_schemas/"),
+    SquemaTypes.explainer: os.path.join(
+        dashai_path, "back/explainability/explainers_schema/"
+    ),
```

Comment:
It seems to me that this is no longer necessary and will be deleted in the next versions, @cristian-tamblay ?